iOS/DuckDuckGo/DaxDialogs.swift

```diff
@@ -686,7 +686,7 @@ private extension ViewHighlighter {
 
 }
 
-#if canImport(XCTest)
+#if DEBUG
```

Comment:
Compilation for this fails on Xcode 26. To make it more reliable I've changed it to a `DEBUG` check.